src/view/uniprotkb/components/XRefView.tsx

```diff
@@ -91,15 +91,14 @@ const XRefCategoryInfoList: React.FC<XRefCategoryInfoListProps> = ({
   const infoData = databases.sort().map(database => ({
     title: database.database,
     content: (
-      <ul className="no-bullet">
-        {database.xrefs.map(xref => (
-          <XRefItem
-            xRefEntry={xref}
-            primaryAccession={primaryAccession}
-            key={v1()}
-          />
-        ))}
-      </ul>
+      <ExpandableList descriptionString={`${database.database} references`}>
```

Comment:
Maybe `links` instead of `references` as the reference word is sometimes used for publication

Reply:
Done